manifests/install.pp

```diff
@@ -1,14 +0,0 @@
-# Private class
-class f5::install {
-  if $::puppetversion and $::puppetversion =~ /Puppet Enterprise/ {
```

Comment:
The better way to handle this would be to leave the f5::install class as many pre-existing users are using this and just comment out the pe_gem dependency.

cc @ericzji

Reply:
Really? The `f5:install` class isn't mentioned in the README and is marked `Private class` on line 1.  I could move the faraday package resource back to this class, but there really did seem little point in keeping it.  I also thought committing commented out code was considered bad practice these days?

Reply:
Yes, the `cisco_ios` module uses this model:

https://github.com/puppetlabs/cisco_ios/blob/master/manifests/init.pp

And the `device_manager` module automatically leverages this model:

https://github.com/puppetlabs/device_manager/blob/master/manifests/init.pp#L76